src/pages/MetricDetail.svelte

```diff
@@ -168,6 +180,40 @@
     <table>
       <col />
       <col />
+      {#if getGlamUrl(selectedAppVariant)}
+        <tr>
+          <td>
+            GLAM
+            <HelpHoverable
+              content={'View this metric in the Glean Aggregated Metrics (GLAM) dashboard'}
+              link={'https://docs.telemetry.mozilla.org/cookbooks/glam.html'} />
+          </td>
+          <td>
+            <AuthenticatedLink href={getGlamUrl(selectedAppVariant)}>
+              {selectedAppVariant.bigquery_names.glam_etl_name}
+            </AuthenticatedLink>
+          </td>
+        </tr>
+      {/if}
+      {#if SUPPORTED_LOOKER_METRIC_TYPES.has(metric.type)}
+        <tr>
+          <td>Looker</td>
+          <td>
+            {#each selectedAppVariant.bigquery_names.stable_ping_table_names as [sendInPing, tableName]}
+              In
+              <AuthenticatedLink
+                href={getLookerExploreURL(params.app, sendInPing.replace(/-/g, '_'))}>
+                {sendInPing}
+              </AuthenticatedLink>
+              as
+              <AuthenticatedLink
+                href={getLookerExploreURL(params.app, sendInPing.replace(/-/g, '_'), metric.type === 'counter' ? params.metric : selectedAppVariant.bigquery_names.metric_table_name, tableName, metric.type === 'counter' && 'metrics')}>
```

Comment:
This is way too much domain-specific business logic for my liking. I'm thinking the right level to do this is in the python script we use to generate the glean metadata.